sdk/identity/identity/src/credentials/visualStudioCodeCredential.ts

```diff
@@ -88,6 +88,9 @@ export class VisualStudioCodeCredential implements TokenCredential {
     } else {
       this.tenantId = CommonTenantId;
     }
+    if (this.tenantId === "adfs") {
+      throw new CredentialUnavailable("The VisualStudioCodeCredential is not supported in Sovereign Clouds")
```

Comment:
This is not about sovereign clouds.
VSCode supports sovereign clouds.

The error message can be changed to say. "The VisualStudioCodeCredential is not supported against ADFS tenant / authorities."

Reply:
Thank you Vinay! I'll update this soon.